.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,12 +8,11 @@ Implements #254
 See also #23
 -->
 
-
 ## Checklist
 
-- [ ] `npm test` passes on your machine
-- [ ] New tests added or existing tests modified to cover all changes
-- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
-- [ ] Related API Documentation was updated
-- [ ] Affected artifact templates in `packages/cli` were updated
-- [ ] Affected example projects in `packages/example-*` were updated
+* [ ] `npm test` passes on your machine
```

Comment:
Hmm, why `*` is better than `-`? Is there a configuration option to override?

Reply:
I've looked into this a while ago, and prettier doesn't have configurable list styling, and it doesn't look to be in scope for a while. https://github.com/prettier/prettier/issues/3025

Reply:
No config option other than to extend the `markdown` parser used by `prettier` and change that there. `*` and `-` are treated the same by GFM (GitHub Flavoured Markdown). I think the `markdown` parser just uses `*` as it's style. 

In VSCode, you can continue using `-` and saving the file will format it to use `*` now.

Reply:
Forcing to use `*` is fairly annoying :-(

Reply:
Yeah, I also like `-` more than `*`. However, I am happy to give up this preference in exchange for automated linting & formatting of Markdown files.

Perhaps we can contribute a feature to prettier to make the preference of `*` vs. `-` configurable? They already allow configuration of the quote symbol used in JavaScript files (`'` or `"`), I think that should serve as a good precedent?